contracts/base/GuardManager.sol

```diff
@@ -2,9 +2,10 @@
 /* solhint-disable one-contract-per-file */
 pragma solidity >=0.7.0 <0.9.0;
 
-import {Enum} from "../common/Enum.sol";
+import {Enum} from "../interfaces/IEnum.sol";
 import {SelfAuthorized} from "../common/SelfAuthorized.sol";
 import {IERC165} from "../interfaces/IERC165.sol";
+import {IGuardManager} from "../interfaces/IGuardManager.sol";
```

Comment:
OCD-nit: Also unrelated to this PR, but maybe we should consider ordering imports for consistency?

Reply:
Quick search didn't come up with anything :disappointed:.

Reply:
I know there's a plugin for typescript: https://github.com/trivago/prettier-plugin-sort-imports

They have multiple pre-processors, so one could extend it with a solidity one on an ecosystem Friday...

Reply:
Apparently it can be used for solidity as well: https://github.com/foundry-rs/foundry/issues/3396

Reply:
Should we add that plugin and try? @nlordell @mmv08

Reply:
Created an issue for later 👍🏾